operators/bosh-operator/DirectorService.js

```diff
@@ -876,6 +876,7 @@ class DirectorService extends BaseDirectorService {
                 }
                 logger.info(`Scheduling Backup for instance : ${this.guid} with backup interval of - ${options.repeatInterval}`);
                 //Even if there is an error while fetching backup schedule, trigger backup schedule we would want audit log captured and riemann alert sent
+                // This flow has to be revisited when we start supporting K8s through service manager
```

Comment:
This flow has to be revisited when we start supporting K8s through service manager @jayeshjk

Reply:
@amitmalav Can you please add same comment at [deleteRestoreFile](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/5179c09b2c1c2859bb752d8fb4309d83aaeb9197/operators/bosh-operator/DirectorService.js#L116)
@jayeshjk Please check this place as well.

Reply:
@subhankarc @amitmalav @kamath-prasad  -- Just a thought, if we also additionally provide backup and restore APIs as admin endpoints, then the same can be invoked via Scheduler and we can all together avoid needing UAA. Thus we can even enable B&R even for K8s?

Reply:
Makes sense. I would propose to relook at all the APIs and redesign them and remove unneeded ones.